modules/claims_api/spec/requests/v1/claims_request_spec.rb

```diff
@@ -72,35 +72,59 @@
+      it 'shows a single Claim through auto established claims', run_at: 'Wed, 13 Dec 2017 03:28:23 GMT' do
+        with_okta_user(scopes) do |auth_header|
+          create(:auto_established_claim,
+                 source: 'abraham lincoln',
+                 auth_headers: { some: 'data' },
+                 evss_id: 600_118_851,
+                 id: 'd5536c5c-0465-4038-a368-1a9d9daf65c9')
+          VCR.use_cassette('evss/claims/claim') do
+            get(
+              '/services/claims/v1/claims/d5536c5c-0465-4038-a368-1a9d9daf65c9',
+              params: nil, headers: request_headers.merge(auth_header)
+            )
+            expect(response).to match_response_schema('claims_api/claim')
+          end
+        end
+      end
+
+      it 'shows a single Claim through auto established claims when camel-inflected',
+         run_at: 'Wed, 13 Dec 2017 03:28:23 GMT' do
+        with_okta_user(scopes) do |auth_header|
+          create(:auto_established_claim,
+                 source: 'abraham lincoln',
+                 auth_headers: { some: 'data' },
+                 evss_id: 600_118_851,
+                 id: 'd5536c5c-0465-4038-a368-1a9d9daf65c9')
+          VCR.use_cassette('evss/claims/claim') do
+            get(
+              '/services/claims/v1/claims/d5536c5c-0465-4038-a368-1a9d9daf65c9',
+              params: nil, headers: request_headers_camel.merge(auth_header)
+            )
+            expect(response).to match_camelized_response_schema('claims_api/claim')
+          end
         end
       end
     end
 
-    it 'shows a single Claim through auto established claims when camel-inflected',
-       run_at: 'Wed, 13 Dec 2017 03:28:23 GMT' do
-      with_okta_user(scopes) do |auth_header|
-        create(:auto_established_claim,
-               auth_headers: { some: 'data' },
-               evss_id: 600_118_851,
-               id: 'd5536c5c-0465-4038-a368-1a9d9daf65c9')
-        VCR.use_cassette('evss/claims/claim') do
-          get(
-            '/services/claims/v1/claims/d5536c5c-0465-4038-a368-1a9d9daf65c9',
-            params: nil, headers: request_headers_camel.merge(auth_header)
-          )
-          expect(response).to match_camelized_response_schema('claims_api/claim')
+    context 'when source does not match' do
+      it 'shows a single Claim through auto established claims', run_at: 'Wed, 13 Dec 2017 03:28:23 GMT' do
+        with_okta_user(scopes) do |auth_header|
+          create(:auto_established_claim,
+                 source: 'oddball',
+                 auth_headers: { some: 'data' },
+                 evss_id: 600_118_851,
+                 id: 'd5536c5c-0465-4038-a368-1a9d9daf65c9')
+          VCR.use_cassette('evss/claims/claim') do
+            get(
+              '/services/claims/v1/claims/d5536c5c-0465-4038-a368-1a9d9daf65c9',
+              params: nil, headers: request_headers.merge(auth_header)
+            )
+
+            # TODO: What is the expectation at this point...?
```

Comment:
do we know the answer as to what to assert yet?

Reply:
shoot! i cant believe i missed that! added an expectation here: https://github.com/department-of-veterans-affairs/vets-api/pull/5285/commits/9abce19acf521d19a60eecb97e8e7d75d35857db